dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/MacCatalystApp1.csproj

```diff
@@ -1,11 +1,63 @@
+         either BOTH runtimes must be indicated or ONLY macatalyst-x64. -->
+    <!-- ex. <RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers> -->
+    <OutputType>Exe</OutputType>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>true</ImplicitUsings>
+  </PropertyGroup>
+
+  <!-- Notes about the values set below:
+    1. For macOS it's possible to sign both the app bundle (EnableCodesigning=true) and the package (*.pkg) (EnablePackageSigning=true),
+       and these are signed separately and with different certificates.
+          CodesignKey: this is the signing key used for the app bundle
+          PackageSigningKey: this is the signing key used for the package
+    2. Publishing to the App Store requires signing both the app bundle and the package.
+          Must be `Apple Distribution: ...' for the app bundle. Note that apps signed like this will not execute locally.
+          They have to be published to the App Store and then downloaded (Apple will resign the app with a different signing identity that allows for local execution).
+          Must be 3rd Party Mac Developer Installer: ... for the pkg
+    3. Publishing outside of the App Store (i.e. only notarizing) requires:
+          Must be Developer ID Application: ... for the app bundle
+          Must be Developer ID Installer: ... for the pkg
+    4. During development, use the Apple Development: ... signing key (typically to verify that the app works when is signed and entitlements are enforced).
+    5. Depending on the entitlements the app needs, a specific provisioning profile (CodesignProvision) might be needed.
+    6. UseHardenedRuntime must be set to true when app sandbox is enabled in info.plist.
+    -->
+  <!--
+  For Debug: (Note: Replace placeholder information before building)
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <EnableCodeSigning>false</EnableCodeSigning>
+    <ProvisionType>Manual</ProvisionType>
+    <CreatePackage>false</CreatePackage>
+    <EnablePackageSigning>true</EnablePackageSigning>
+    <CodesignKey>Apple Distribution: YOURNAME (*******)</CodesignKey>
+    <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
+    <CodesignProvision>YOUR PROFILE NAME</CodesignProvision>
+    <PackageSigningKey>3rd Party Mac Developer Installer: YOURNAME (*******)</PackageSigningKey>
+    <UseHardenedRuntime>false</UseHardenedRuntime>
+  </PropertyGroup>
+  -->
+  <!--
+  For Release: (Note: Replace placeholder information before building)
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <MtouchLink>Full</MtouchLink>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">MacCatalystApp1</RootNamespace>
     <OutputType>Exe</OutputType>
     <Nullable>enable</Nullable>
     <ImplicitUsings>true</ImplicitUsings>
     <SupportedOSPlatformVersion>minOSVersion</SupportedOSPlatformVersion>
```

Comment:
These properties (RootNamespace, OutputType, Nullable, ImplicitUsings, SupportedOSPlatformVersion) should go in the very top PropertyGroup (without any Condition), and they should *not* be commented out.